src/views/product-integration/component-view/components/searchable-variable-group-list/style.module.css

```diff
@@ -19,7 +19,7 @@
 }
 
 .results {
-	color: var(--gray-3);
+	color: var(--token-color-foreground-faint);
```

Comment:
@braydenlove found one more 🙃 Let me know if this is the right value. I couldn't find this tag in figma to see what it should be.

Reply:
We have a [ticket](https://app.asana.com/0/1203792616809419/1203988523494882/f) to shore this up from a typography standpoint, the color you chose is perfect!

Reply:
Okay I can add those type updates here then.